packages/block-library/src/navigation-link/editor.scss

```diff
@@ -83,6 +82,13 @@
 	.block-editor-block-toolbar {
 		left: $block-padding;
 	}
+
+	// Hide the draggable mover until functionality for positioning it on drag correctly is added
+	.block-editor-block-toolbar .components-toolbar.block-editor-block-mover {
```

Comment:
I was able to reduce this to 

```scss
[data-type="core/navigation-link"] {
	.block-editor-block-toolbar {
		left: $block-padding;
	}

	// Hide the draggable mover until functionality for positioning it on drag correctly is added
	.block-editor-block-mover__control-drag-handle {
		display: none;
	}
}
```

...and still get the same effect. Did I miss something or can we greatly reduce the specificity here?

Reply:
I had to up the specificity because of this monster statement:

```
.block-editor-block-contextual-toolbar:not([data-align="wide"]):not([data-align="full"]) .block-editor-block-mover:not(.is-horizontal) .block-editor-block-mover__control
```

<img width="1398" alt="Screen Shot 2020-01-16 at 8 04 40 AM" src="https://user-images.githubusercontent.com/967608/72531354-e20b4400-3836-11ea-8a1e-d0f9ec80e4b5.png">

Reply:
Ah I see. Must have not been applied on my viewport size. 👍